sonarqube-companion-rest/src/main/java/pl/consdata/ico/sqcompanion/config/AppConfigConfiguration.java

```diff
@@ -66,14 +69,9 @@ public AppConfig appConfig(final ObjectMapper objectMapper) throws IOException {
         module.addDeserializer(NoImprovementWebhookActionData.class, new NoImprovementWebhookActionDataDeserializer());
         objectMapper.registerModule(module);
 
-        if (!appConfigPath.toFile().exists()) {
-            log.info("App configuration not exist, creating default [path={}]", appConfigPath);
-            objectMapper.writeValue(appConfigPath.toFile(), getDefaultAppConfig());
-        }
-
-        final AppConfig appConfig = objectMapper.readValue(appConfigPath.toFile(), AppConfig.class);
+        final AppConfig appConfig = appConfigStore.read(objectMapper, getDefaultAppConfig());
         log.info("App config loaded [appConfig={}]", appConfig);
-        store(objectMapper, appConfig);
+        appConfigStore.store(objectMapper, appConfig);
```

Comment:
po co jest ten store po wczytaniu?

Reply:
ten zapis wcześniej dopieszczał istniejący config(uzupełniał uuid itd), ale jak tak teraz patrze to nie jest zaczytywany ponownie więc ta linia nie ma sensu :)

Reply:
"dopieszczał config" - podoba mi się :D
czyli zostaje, zobacz że nie musi zczytywać ponownie bo przecież zwraca ten właśnie uaktualniony model który tylko serializowaliśmy. czyli w returnie leci ten dopieszczony, imo to jest git chociaż trochę nieoczywiste. może chociaż jakiś doc?

Reply:
Wywaliłem to. Te custome deserializery dopieszczają config więc w runtime mamy wszystko dopieszczone. Przy pierwszej zmianie settingsów to się zapisze na stałe.